package.json

```diff
@@ -32,12 +32,15 @@
     "@nestjs/mapped-types": "*",
     "@nestjs/passport": "^10.0.3",
     "@nestjs/platform-express": "^10.0.0",
+    "@supabase/supabase-js": "^2.46.1",
```

Comment:
Why supabase?

Reply:
I think that we agreed on either minIO or local image storage. What are the limits of the Supabase's S3?